### PR TITLE
Remote control

### PR DESCRIPTION
**Pull request by Ongy**
Adds new API functions to allow remote controlling another session.

Implemented for and tested with https://github.com/home-assistant/core/pull/76801

### CODE REVIEW

File: jellyfin_apiclient_python/api.py [hunk 1]
```diff
@@ -382,6 +401,56 @@ def session_progress(self, data):
     def session_stop(self, data):
         return self.sessions("/Playing/Stopped", "POST", json=data)
 
+    def remote_pause(self, id):
+        return self.remote(id, "Pause")
+
+    def remote_playpause(self, id):
+        return self.remote(id, "PlauPause")
```
Comment (iwalton3) [inline jellyfin_apiclient_python/api.py:408]: Typo?
```suggestion
        return self.remote(id, "PlayPause")
```

**Review by iwalton3 — CHANGES_REQUESTED** [review]

**Comment by Ongy** [comment]
Rebased for Readme.md conflict.

**Review by iwalton3 — APPROVED** [review]

**Comment by iwalton3** [comment]
https://pypi.org/project/jellyfin-apiclient-python/1.9.0/

Thanks for contributing to Home Assistant!

**Comment by iwalton3** [comment]
Looks like a broken type hint was causing some issues. Should be fixed now.

See: https://github.com/jellyfin/jellyfin-mpv-shim/issues/292

https://pypi.org/project/jellyfin-apiclient-python/1.9.1/

**Pull request merged**